api/raft.proto

```diff
@@ -16,6 +16,13 @@ service Raft {
 		option (docker.protobuf.plugin.tls_authorization) = { roles: "swarm-manager" };
 	};
 
+	// RaftMessageStream accepts a stream of raft messages to be processed on a raft member, 
+	// returning a ProcessRaftMessageResponse when processing of the streamed messages is complete.
+	// It is called from the Raft leader, which uses it to stream messages to a raft member.
```

Comment:
Explain here that a single stream call corresponds to a single, disassembled message.